addon/components/frost-list-toggle.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember'
+const {Component} = Ember
+import layout from '../templates/frost-list-toggle'
+
+export default Component.extend({
```

Comment:
I think we had discussed naming this `expansion` instead of `toggle` before

e.g. `controls.expansion`, since it's not really a toggle and expansion is a pretty clear tie to expand/collapse